wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -38,7 +38,7 @@ Rule proposals as well as suggestions are discussed in the [Ranking Criteria Sub
 - **Minijack**: Two consecutive notes in the same column, usually pressed at a faster pace compared to surrounding notes.
 - **Release**: The end of a long note which must be released within a certain timing  window to score the entirety of the note properly.
 - **Roll**: A variation of a stream consisting of notes that go from the first to the last column or last to the first column, usually done in quick succession.
-- **Shield**: A note that appears in the same column immediately after the release of a long note.
+- **Shield**: A note that appears in the same column immediately before or after the release of a long note.
```

Comment:
don't know a lot about mania but I'm pretty sure this is wrong

Reply:
this is right , may and / or better  ?  
referance : https://i.ppy.sh/e7ceb44562f4564affa4b964e19259f9d0e3cfa1/68747470733a2f2f7075752e73682f434831454c2f376666393062383835612e706e67

Reply:
if reverse shields are included
```suggestion
- **Shield**: A note that appears in the same column immediately before a long note or after the release of a long note.
```